### ISSUE TITLE
Pharo3 vm's SSL lib is too outdated (SSL23_GET_SERVER_HELLO:tlsv1 alert protocol version)

### ISSUE THREAD

**Issue by jbrichau** [GsDevKit/GsDevKit_home#221]
Installation of the server failed for me on centos 7 when trying to run the `createStone` script.

The problem is in the step that creates a Pharo3 image and needs SSL connection to github to download some projects. The SSL library coming with the Pharo3 vm is just too outdated and you get the `SSL23_GET_SERVER_HELLO:tlsv1 alert protocol version`. More transcript context in the snippet below.

You can workaround this by replacing the SSL plugin in the vm with a more recent version:
```
wget https://github.com/squeak-smalltalk/squeakssl/releases/download/v0.2.0b/squeakssl_v0.2.0b.zip
unzip squeakssl_v0.2.0b.zip
mv squeakssl_v0.2.0b/linux32/so.SqueakSSL $GS_HOME/shared/pharo/pharo-vm/libSqueakSSL.so
```
=================
Downloading GemStone/S 3.4.0
/home/vagrant/GsDevKit_home/shared/downloads//products/GemStone64Bit3.4.0-x86_64.Linux already exists to replace it, remove or rename it and rerun this script :: downloadGemStone 3.4.0
=================
  GsDevKit script: create_gs_server_image -d /home/vagrant/GsDevKit_home/shared/pharo gsDevKitCommandLine
             path: /home/vagrant/GsDevKit_home/shared/bin/create_gs_server_image
=================
create gsDevKitCommandLine.image image using /home/vagrant/GsDevKit_home/sys/default/pharo/gsDevKitCommandLineLoad.st

Fetched -> BaselineOfMetacello-EstebanLorenzano.97 --- filetree:///home/vagrant/GsDevKit_home/shared/repos/metacello-work/repository --- filetree:///home/vagrant/GsDevKit_home/shared/repos/metacello-work/repository
Loaded -> BaselineOfMetacello-EstebanLorenzano.97 --- filetree:///home/vagrant/GsDevKit_home/shared/repos/metacello-work/repository --- filetree:///home/vagrant/GsDevKit_home/shared/repos/metacello-work/repository
Loading baseline of BaselineOfMetacello...16694:error:1407742E:SSL routines:SSL23_GET_SERVER_HELLO:tlsv1 alert protocol version:s23_clnt.c:583:

...RETRY->BaselineOfFileTree16694:error:1407742E:SSL routines:SSL23_GET_SERVER_HELLO:tlsv1 alert protocol version:s23_clnt.c:583:

...RETRY->BaselineOfFileTree16694:error:1407742E:SSL routines:SSL23_GET_SERVER_HELLO:tlsv1 alert protocol version:s23_clnt.c:583:
'Errors in script loaded from /home/vagrant/GsDevKit_home/sys/default/pharo/gsDevKitCommandLineLoad.st'
ConnectionClosed: Connection closed while waiting for data.
Socket>>waitForDataFor: in Block: [ ConnectionClosed signal: 'Connection closed whil...etc...
Socket>>waitForDataFor:ifClosed:ifTimedOut:
Socket>>waitForDataFor:
```

**Comment by dalehenrich** [comment]
Thanks @jbrichau ... I guess I can add that to the pharo3 download/build script ... I hoping to carve out a little bit of time to work on GsDevkit_home ... soon:)

**Comment by dalehenrich** [comment]
fixed with pr #283
